### ISSUE TITLE
[Enhancement] New email notification for Approved status

### ISSUE THREAD

**Issue by patrickm02L** [cityofaustin/atd-data-tech#14994]
## Description
broken out of #14514

> 1. Need to develop a procedure for emailing people once a decision is made on whether the application is accepted or deferred.

**Comment by patrickm02L** [comment]
## 1/12
In LS Sync, the team is going to manually send emails for now, but would like to prioritize this functionality for the next round - estimated Feb to coincide with new applications that would come in March - this is contingent on funding being.
